app/frontend/src/aquifers/components/Search.vue

```diff
@@ -313,8 +313,14 @@ export default {
             aquifer.gs = geomCache[aquifer.id]
           })
           this.response = response.data
-          this.scrollToTableTop()
           this.loading = false
+          if (response.data.results.length > 0) {
```

Comment:
Seems like this is assigned to `this.aquifers_search_results` above, might be cleaner to change it to
```
if (this.aquifers_search_results.length > 0) {
```